### PR TITLE
doc/radosgw: multisite - edit "functional changes"

### PR DESCRIPTION
**Pull request by zdover23**
Improve the wording of the section "Functional Changes from Infernalis" in doc/radosgw/multisite.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -71,15 +71,16 @@ Functional Changes from Infernalis
 ==================================
 
 Beginning with Kraken, each Ceph Object Gateway can be configured to work in an
-active-active zone configuration, allowing for writes to non-master zones.
+active-active zone configuration. This makes it possible to write to non-master
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:74]: At both of these loci I think 'mode' fits better than 'configuration', ymmv.
Non-blocking.

**Review by anthonyeleven — APPROVED** [review]
see comment

File: doc/radosgw/multisite.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/multisite.rst:74]: I'll chew on it for a day while I make other small alterations like this one. My very first thought is that "mode" is a word pretty much claimed by vi and vim, but my very first thought has been wrong before and you do know the pragmatics better than I.

Comment (anthonyeleven) [reply doc/radosgw/multisite.rst:74]: "Configuration" to me is something static:  `ceph.conf`, the mon's central config DB, even runtime injections.  "Mode" connotes the fashion in which things happen dynamically, like the two weeks I once spent in night mode, or the occasions when I'm in single-parent mode.

That said, use by `vi` alone is reason to chose a different word ;)

>"Droids, I want to lose.  I mean, I want
to lose.  I want to see line editors on CRTs and nulls in my files.

Comment (zdover23) [reply doc/radosgw/multisite.rst:74]: 1. I think I'm convinced.
2. That is an obscure Alice's Restaurant-adjacent reference. You are the James Joyce of the Ceph documentation.
3. Are we on different text-editor-based sides of a religious war?

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50277 - Quincy backport
https://github.com/ceph/ceph/pull/50278 - Pacific backport

**Pull request merged**